internal/expr/query.go

```diff
@@ -8,36 +8,38 @@ import (
 	"strings"
 )
 
-func (qe *QueryExpr) QueryArgs() []any {
-	return qe.args
+// PrimedQuery contains all concrete values needed to run a SQLair query on a
+// database.
+type PrimedQuery struct {
```

Comment:
I had a thought that `PrimedQuery` is not such a great name due to being a member of `sqlair.Query`, but I think it's OK, because it is distinct by being from the `expr` package, from who's point-of-view the work is done.

Reply:
I'm not 100% happy with it either, but it can stay there for now, if we think of a better name we can change it